<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatPerspectiveView</class>
 <widget class="QWidget" name="MatPerspectiveView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Fov</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Near</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Far</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="fov">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
       <property name="value">
        <double>60.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="width">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.000000000000000</double>
       </property>
       <property name="value">
        <double>1280.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="height">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.000000000000000</double>
       </property>
       <property name="value">
        <double>720.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="nearDist">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="minimum">
        <double>-999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.000000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="farDist">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="maximum">
        <double>999999.000000000000000</double>
       </property>
       <property name="value">
        <double>99.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
